core/src/smartcontracts/isi/world.rs

```diff
@@ -368,6 +368,35 @@ pub mod isi {
             })
         }
     }
+
+    impl Execute for SetParameter<Parameter, Account> {
+        type Error = Error;
+
+        #[metrics(+"set_parameter")]
+        fn execute(
+            self,
+            _authority: <Account as Identifiable>::Id,
+            wsv: &WorldStateView,
+        ) -> Result<(), Self::Error> {
+            let new_param = self.parameter.clone();
```

Comment:
You can use destructuring instead of clone here:

```suggestion
            let SetParameter { parameter, .. } = self;
```